### DISCUSSION TITLE
Open Source Projects

### DISCUSSION THREAD

**Discussion by jurmy24** [community/community#21551] [New to GitHub]
<p>Anyone know any beginner friendly open source projects?</p>

**Comment by vamche** [comment, marked as answer]
<p>Did you try <a href="http://up-for-grabs.net/#/" rel="nofollow noopener">http://up-for-grabs.net/#/</a> ? </p>

**Comment by jurmy24** [comment]
<p>Yes I found this website right after (: thanks anyway</p>
